### PR TITLE
Gets the localisation strings from the print config

### LINKED ISSUE #1905: Add print template to i18n strings
Update the Lingua extractors:
https://github.com/camptocamp/c2cgeoportal/blob/master/setup.py#L124-L127

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1905

### CODE REVIEW

File: c2cgeoportal/lib/lingua_extractor.py [hunk 1]
```diff
@@ -68,20 +69,36 @@ def __call__(self, filename, options):
...
+                        (filename, u"template/%s" % template)
+                    ))
+                    result += [
+                        Message(
+                            None, attribute, None, [], u"", u"",
+                            (filename, u"template/%s/%s" % (template, attribute))
+                        )
+                        for attribute in config.get("templates")[template].attributes.keys()
+                    ]
+                return result
```
Comment (pvalsecc) [inline c2cgeoportal/lib/lingua_extractor.py]: Maybe you don't want to silently ignore files you don't understand:
```
else:
    goBoom!
```

File: c2cgeoportal/lib/lingua_extractor.py [hunk 2]
```diff
@@ -68,20 +69,36 @@ def __call__(self, filename, options):
...
             config = yaml.load(config_file)
-            return [
-                Message(
-                    None, raster_layer, None, [], u"", u"",
-                    (filename, u"raster/%s" % raster_layer)
-                )
-                for raster_layer in config.get("raster", {}).keys()
-            ]
+            if "raster" in config:
+                return [
```
Comment (pvalsecc) [inline c2cgeoportal/lib/lingua_extractor.py]: `# .build/config.yaml`

File: c2cgeoportal/lib/lingua_extractor.py [hunk 3]
```diff
@@ -68,20 +69,36 @@ def __call__(self, filename, options):
...
+                    Message(
+                        None, raster_layer, None, [], u"", u"",
+                        (filename, u"raster/%s" % raster_layer)
+                    )
+                    for raster_layer in config.get("raster", {}).keys()
+                ]
+            elif "templates" in config:
+                result = []
```
Comment (pvalsecc) [inline c2cgeoportal/lib/lingua_extractor.py]: `# print config`

**Comment by sbrunner** [comment]
Thanks @pvalsecc fixed :-)

**Pull request merged**